app/components/Event.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import styles from './event.scss';
+var EventPanel = require('./Panel.jsx')
+//import { Panel } from 'react-bootstrap';
+require('bootstrap/dist/css/bootstrap.css') ;
```

Comment:
Shouldn't need this line anymore. The one in App.js should apply to the entire project